Lib/axisregistry/data/y_vertical_extension.textproto

```diff
@@ -0,0 +1,16 @@
+# YEXT based on https://github.com/TypeTogether/Playwrite
+tag: "YEXT"
+display_name: "Vertical Extension"
+min_value: 0
+default_value: 0
+max_value: 100
+precision: 0
+fallback {
+  name: "Default"
+  value: 0
+}
+fallback_only: false
+description: "The axis extends glyphs in the Y dimension, such as the
+  Cap Height, Ascender and Descender lengths. This is a relative axis,
+  starting at 0% and going to the typeface's individual maximum
+  extent at 100%."
```

Comment:
Each line needs quote marks e.g https://github.com/googlefonts/axisregistry/blob/main/Lib/axisregistry/data/bleed.textproto#L14-L16

Reply:
Augh! I forgot the quotes! Thanks, Marc